manifests/type/helm/storage.yaml

```diff
@@ -39,7 +39,8 @@ spec:
                       "type": "object",
```

Comment:
As discussed previously, please move this file to the `manifests/type/helm/release/storage.yaml`. 
The #63 bases on your changes, but I also move this file to have a bit cleaner directory structure (as we have two Helm-related backends).


See the change in this file on my PR: https://github.com/capactio/hub-manifests/pull/63/files#diff-ed1b6657ca8ae4824de4d8127a44b0dd498f59ac60917bfc0b758c8fb12c0f24
+ then the `passing-b.yaml` Impl needs to be changed: https://github.com/capactio/hub-manifests/pull/63/files#diff-2764b7e575f40db2193141732af4a605af5a396ad85e5d49ad2f441c05677de1

Reply:
I went with 1 👍